src/main/java/com/easypost/model/Shipment.java

```diff
@@ -1012,9 +1039,25 @@ public Shipment buy(final Rate rate, final boolean withCarbonOffset) throws Easy
      * @param apiKey API key to use in request (overrides default API key).
      * @return Shipment object
      * @throws EasyPostException when the request fails.
+     *
+     * When Java Client Library rewrite happens, the apiKey param will be replaced with endShipperId
```

Comment:
Issue: This must be a TODO comment to signify an action is required from us. This should also be in the code itself and not the docstring since the docstring is viewable by users and this note is only intended for us.

Reply:
fixed